packages/authenticator/amplify_authenticator/example/lib/main.dart

```diff
@@ -83,7 +83,7 @@ class _MyAppState extends State<MyApp> {
     // automatically configures the default for the others.
     const stringResolver = AuthStringResolver(
       buttons: LocalizedButtonResolver(),
-      countries: LocalizedCountryResolver(),
+      areas: LocalizedDialResolver(),
```

Comment:
I don't think `areas` is correct. See https://github.com/aws-amplify/amplify-flutter/pull/3354#discussion_r1267128723.